gesund/metrics/classification/create_validation.py

```diff
@@ -127,45 +127,63 @@ def load(self, validation_collection_data, class_mappings, filtering_meta=None):
         
         return overall_metrics
     
-    def plot_metrics(self, metrics, jsons_dir, plot_dir):
+    def plot_metrics(self, metrics, json_outputs_dir, plot_outputs_dir, plot_configs):
```

Comment:
please lets add variable typing as well for better code clarity 
https://realpython.com/python-type-checking/

Reply:
thanks updated.